### ISSUE TITLE
Remove Mapbox Token online verification function

### ISSUE THREAD

**Issue by lujieha** [maps-gl/maps-gl#2]
Remove Mapbox Token online verification function

**Comment by stevage** [comment]
Is your intention to make this fork unable to use Mapbox services?

**Comment by fakegis** [comment]
Yes, it becomes pure WebGL rendering code, which can integrate more services such as google maps in the future

**Comment by stevage** [comment]
Wait, you want it to support Google Maps, but not support Mapbox?

**Comment by zakjan** [comment]
@stevage If I'm not mistaken, Mapbox support needs only access token in the URL, or no? It would be similar as Maptiler support - no specific param, just url.

**Comment by stevage** [comment]
So you want to remove the whole `mapbox://` URL structure as well? I was imagining going the other direction, and maybe supporting things like `maptiler://` or something.

I really suggest taking a moment to think through some of these decisions rather than rushing through them. I don't think there's any urgency on this stuff.

**Comment by zakjan** [comment]
Agree about no urgency. I just stumbled upon this thread. However, in my opinion such protocols are unnecessary.
